api/client/cluster/services.go

```diff
@@ -0,0 +1,74 @@
+package cluster
+
+import (
+	"github.com/libopenstorage/openstorage/services"
+)
+
+// ServiceEnterMaintenanceMode takes node out of cluster
+func (c *clusterClient) ServiceEnterMaintenanceMode(exitOut bool) error {
+	req := c.c.Put().Resource(clusterPath + ServicePath + "/entermaintenance")
+	res := req.Do()
+	if res.Error() != nil {
+		return res.FormatError()
+	}
+
+	return nil
+}
+
+// ServiceExitMaintenanceMode puts node back into cluster
+func (c *clusterClient) ServiceExitMaintenanceMode() error {
+	req := c.c.Put().Resource(clusterPath + ServicePath + "/exitmaintenance")
+	res := req.Do()
+	if res.Error() != nil {
+		return res.FormatError()
+	}
+
+	return nil
+}
+
+// ServiceAddDrive adds specified drive
+func (c *clusterClient) ServiceAddDrive(op, drive string, journal bool) (string, error) {
```

Comment:
What is op? Is it needed?

Reply:
it's for start operation/status of current operation